### PR TITLE
#161966593 Tests for authentication app

### PR DESCRIPTION
**Pull request by TeamoreA**
#### What does this PR do?
Tests for authentication app
#### Description of Task to be completed?
write separate files for login and registration. Also, write the base file for configurations of tests throughout the app
#### How should this be manually tested?
To test the application follow the steps below.
 - Clone the repo using `git clone https://github.com/andela/ah-technocrats.git`
 - `cd ah-technocrats`
 - `git checkout ch-initial-tests-161966593` to switch to the tests branch
 - Run `virtualenv env -p python3` to create a virtual environment for the application
 - Activate the environment `source env/bin/activate`
 - Install required modules  using `pip install -r requirements.txt`
 - Run `coverage run --source=. manage.py test ` to test
- Then run `coverage report` to get the coverage
#### Any background context you want to provide?
- The project requires that you have python version 3 installed. For the Django version, the project has just been tested with Django 2.1 which is the latest version as of this PR.
#### What are the relevant pivotal tracker stories?
[#161966593](https://www.pivotaltracker.com/n/projects/2225606/stories/161966593)

### CODE REVIEW

**Comment by salma-nyagaka** [comment]
The links seem not to be working

**Comment by TeamoreA** [comment]
> The links seem not to be working, check on that

do you mean the link to the PT story

**Review by verenceLola — COMMENTED** [review]
Good work. 
Can you please add The Checklist section in your PR.
and also, would you include the CI badges and the Coverall budges in your PR markdown.

**Review by SilasKenneth — DISMISSED** [review]

**Review by SilasKenneth — DISMISSED** [review]

**Comment by Mnickii** [comment]
Rebase dev

**Review by SilasKenneth — DISMISSED** [review]

**Comment by kipropbrian** [comment]
This PR has commits from other branches. something funny is going on. Please check.

File: README.md [hunk 1]
```diff
@@ -1,20 +1,38 @@
 ![license](https://img.shields.io/github/license/mashape/apistatus.svg)
-
 [![Build Status](https://travis-ci.org/andela/ah-technocrats.svg?branch=develop)](https://travis-ci.org/andela/ah-technocrats)
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-technocrats/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-technocrats?branch=develop)
```
Comment (Mnickii) [inline README.md]: These changes are out of scope of your PR and story.

**Review by Mnickii — CHANGES_REQUESTED** [review]
Make sure the only changes in this PR are related to your pivotal story.

File: README.md (hunk 1 above)
Comment (TeamoreA) [reply README.md]: Fixed it

**Comment by Mnickii** [comment]
Rebase

File: authors/apps/authentication/models.py [hunk 2]
```diff
@@ -122,5 +122,3 @@ class BlacklistedToken(models.Model):
 
     class Meta:
         db_table = "blacklist"
-class BlacklistedToken(models.Model):
```
Comment (Mnickii) [inline authors/apps/authentication/models.py:125]: Why did you remove this? Do we not need it anymore?

File: requirements.txt [hunk 3]
```diff
@@ -23,14 +23,5 @@ simplejson==3.16.0
 six==1.11.0
 uritemplate==3.0.0
 urllib3==1.24.1
-Django==2.1
```
Comment (Mnickii) [inline requirements.txt:26]: Why did you remove these packages? I'm sure alot of functionality depends on them

**Review by Mnickii — CHANGES_REQUESTED** [review]
If these changes are as a result of an incorrect rebase, please redo the rebase from develop branch

File: authors/apps/authentication/models.py (hunk 2 above)
Comment (TeamoreA) [reply authors/apps/authentication/models.py:125]: The base branch(develop) had the class without the Meta class and the token property which during conflict someone forgot to keep the new version of the class and kept both. So I had to remove it

File: requirements.txt (hunk 3 above)
Comment (TeamoreA) [reply requirements.txt:26]: it was duplicate

**Comment by Mnickii** [comment]
Rebase

**Review by Mnickii — APPROVED** [review]
